.coveragerc

```diff
@@ -723,6 +723,8 @@ omit =
     homeassistant/components/pvoutput/sensor.py
     homeassistant/components/pyload/sensor.py
     homeassistant/components/qbittorrent/sensor.py
+    homeassistant/components/qbittorrent/const.py
```

Comment:
Does const.py need to be excluded? I would expect const.py to not have any executable lines the coverage validator would complain about.

Reply:
It is indeed not necessary, I think I've added it to be sure